rest-fetcher/build.gradle

```diff
@@ -20,15 +21,23 @@ android {
     }
 }
 
+repositories {
+    mavenCentral()
+}
+
 dependencies {
     implementation project(':status-resolver')
 
-    implementation 'com.squareup.retrofit2:retrofit:2.5.0'
-    implementation 'com.squareup.retrofit2:converter-gson:2.3.0'
-    implementation "com.squareup.retrofit2:adapter-rxjava2:2.5.0"
-    implementation 'com.squareup.okhttp3:logging-interceptor:3.12.0'
+    implementation 'com.squareup.retrofit2:retrofit:2.7.2'
+    implementation 'com.squareup.retrofit2:converter-gson:2.4.0'
+    implementation 'com.squareup.okhttp3:logging-interceptor:4.4.0'
 
-    testImplementation 'junit:junit:4.12'
+    testImplementation 'junit:junit:4.13'
+    testImplementation "com.google.truth:truth:1.0.1"
+    testImplementation "com.squareup.okhttp3:mockwebserver:4.7.2"
+    implementation "androidx.core:core-ktx:+"
```

Comment:
nope. Use fixed version. Even AS must shame you for that.

Reply:
1625e84cb028ce7b65cc6bd34201660fe27780f8